server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/modals.ftl

```diff
@@ -45,6 +45,10 @@
                 <div class="col-sm-4 text-right">Instance&nbsp;Id</div>
                 <div class="col-sm-6 text-left">${instance_id}</div>
               </div>
+              <div class="row">
+                <div class="col-sm-4 text-right">Zookeeper&nbsp;Hosts</div>
```

Comment:
Trying to keep the capitalization consistent with how the ZooKeeper project presents it on their website, the `K` should be uppercase.

```suggestion
                <div class="col-sm-4 text-right">ZooKeeper&nbsp;Hosts</div>
```